website/docs/about/contribution.md

```diff
@@ -13,8 +13,8 @@ At this time we are not accepting external contributions from non-HashiCorp empl
 
 While we don’t accept direct contributions of new components or patterns, we do welcome contributions in the following forms:
 
-* [Reporting](https://github.com/hashicorp/design-system/issues/new/choose) or [fixing](#code-contributions) bugs and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
-* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
+* [Reporting](support#file-an-issue) or [fixing](#code-contributions) bugs and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
```

Comment:
The "fixing" link doesn't really do anything because the page isn't long enough for the proper jump, so I've added an additional indicator of the section to refer to. I also found it weird that the "feedback" one was lumped together with the other two.

```suggestion
* [Reporting](support#file-an-issue) or fixing an issue or bug (see [Code contributions](#code-contributions))
* [Providing general feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
```